tests/test_xopen.py

```diff
@@ -566,3 +566,67 @@ def test_xopen_zst_fails_when_zstandard_not_available(monkeypatch):
     with pytest.raises(ImportError):
         with xopen.xopen(TEST_DIR / "file.txt.zst", mode="rb", threads=0) as f:
             f.read()
+
+
+# tests for passing in filehandles
+def test_passing_in_gz_filehandles_for_reading():
```

Comment:
Please also use the imperative mood (just  "pass", not "passing") for consistency with how the other tests are named.